### DISCUSSION TITLE
Multiple "in" Filters

### DISCUSSION THREAD

**Discussion by Samhamsam** [supabase/supabase#4485] [Questions]
I want to do somethink like this:
const cattleIdList = [1,2,3,4]

const { data:, error } = await this.db.from(this.table).select().eq("user_id", await this.getUserId())
                                   .in("father", cattleIdList).in("mother", cattleIdList)

This does not work. I need a unique list. How could i do it?

**Comment by soedirgo** [comment]
Can you elaborate what you're trying to do, together with expected and actual result?

**Reply by Samhamsam** [reply]
So for example:
if i have this data:

```
{id: 11, mother:1, father:5},
{id: 12, mother:2, father:5},
{id: 13, mother:3, father:5}
```
i need:
```
{1:
 [{id: 11, mother:1, father:5}],
},
{2:
 [{id: 12, mother:1, father:5}],
},
{3:
 [{id: 13, mother:1, father:5}],
},
{5:[
{id: 11, mother:1, father:5},
{id: 12, mother:2, father:5},
{id: 13, mother:3, father:5}
]}
```
So short: i need a list of all the kids for every parent. What is the shortest way to do it in Supabase?

My idea was to have two "in" Filters. But this does not work. What i could do would be to get all kids for mother, and all kids for father, make a unique list with javascript and process it further for my end result.
